federation/src/graphql/api/GetTestApiResult.ts

```diff
@@ -1,6 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Same question as the unit-test file

Reply:
here i have no answer why, but my IDE give this hint...